datascience/tables.py

```diff
@@ -1,5 +1,12 @@
 """Tables are sequences of labeled columns."""
 
+import datascience.predicates as _predicates
```

Comment:
Please avoid making unnecessary changes, like re-ordering imports.  We would prefer to see a clean diff that only changes the parts that you modified.